go.sum

```diff
@@ -0,0 +1,377 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Ignore for now, for a future investigation / clean-up: I'm surprised by the number of indirect dependencies.